### DISCUSSION TITLE
Zu wenig Staking erhalten

### DISCUSSION THREAD

**Discussion by kjklj** [iotaledger/firefly#3956] [Help & Support] [answered]
Hallo

Ich stake nun seit 90 Tagen und hab nur ca. die Hälfte erhalten was in den ersten 90 Tagen. Hat sonst noch jemand das gleiche Problem? Hat jemand eine Info dazu?

**Comment by MiKeF05** [comment, marked as answer]
Das ist korrekt. Im zweiten Stakingevent gibt es nur die Hälfte an Rewards.
